docs/Conference-PoS-guide.md

```diff
@@ -0,0 +1,163 @@
+- Click "**Save**" button
+
+### 4. Setup Point of Sale (PoS)
+- Left sidebar under "Plugins", click on "**Point of Sale**"
+- "**App Name**": enter the same merchant name as for the store
+- Click "**Create**" button
+- Now on PoS settings, make sure "**App Name**" and "**Display Title**" are filled out
+- "**Choos Point of Sale Style**": select "**Keypad**"
+- "**Currency**", select the same currency as your store
+- Click on "**Save**" button
+- On the top right click on the "**View**" button and make sure the keypad is shown
+
+### 5. Put PoS link and labels on Bitcoinize device
+
+- Go back to PoS settings in your browser and click on the "**QR-code icon**"
+- On your Bitcoinize (or other) device, open the "**Camera**" app
+- Scroll to the right until you find "**more**" category -> select "**QR-Code**"
+- Now scan the QR-Code shown in your browser (on the PoS settings page)
+- After scanning open the URL in the Chrome browser
+- Check that you see the keypad and the correct merchant name
+- Tap on the 3 dots "**...**" on the top right and select "**Add to home**"
+- Place the icon on the main home screen for easy access
+- Label the device and box with stickers bearing the merchant name
+
+***Testing the PoS device payments and give permissions:***
+- Start the PoS page from the home screen
+- Ensure that the sound on the device is turned up so that there is audio feedback on payment, especially if Bolt Cards are used
+- Enter 0.01 USD (or other currency equivalent) and tap on "**Charge**" button
+- Only first time the browser will ask for NFC permission, tap on the button to give it the requested permission
+- Pay the invoice either with a Bolt Card or Lightning wallet
+- Ensure you hear the sound after the payment
+- Tap on "**View receipt**" button, test printing the receipt by selecting the **POSPrinter** from the dropdown, tap on the "**Print**" button
+
+### 6. Give merchants access to the payment history (optional)
+
+Optionally you can also create a login for each store/merchant on the PoS device so they can access the payment history. This is helpful to double-check what was the last payment or if a payment was already made. You can do so by adding a "Merchant" role with the following permissions:
+- btcpay.store.canmodifyinvoices
+- btcpay.store.canviewstoresettings
+- btcpay.store.canviewpaymentrequestes
+- btcpay.store.canarchivepullpayments
+- btcpay.store.cancreatenonapprovedpullpayments
+
+After that create a user for each store and assign them to the right store.
+
+## Setup a Bolt Cards provider
+
+We will create a separate store that is serving as a Bolt Cards provider. To find it easier in the list of stores you can append the store name with a "z", e.g. "z - Bolt Cards Provider".
+
+For this special store we will connect a Blink account with the important differences compared to merchant Blink accounts:
+- the API key has to have also "write" permission, otherwise Bolt Cards won't be able to pull the funds
```

Comment:
Just emphasising more with obld
```suggestion
- the API key has to have also **write permission**, otherwise Bolt Cards won't be able to pull the funds
```